route/index.js

```diff
@@ -1,9 +1,10 @@
 const express = require('express');
-const router = express.Router();
+const indexRouter = express.Router();
+const puppyRouter = express.Router();
 
-const Puppy = require('../db/model/puppies');
+const { Puppy } = require('../db/model/puppies');
```

Comment:
your `puppies` module exports an object with `{ db, Puppy }`. If you just `const Puppy = require(..)` that module, you'll get the _object_, not the `Puppy` part of that object. I use destructuring here to get access to the model itself, which is what we need. Without this, the routes didn't actually work.

This might have originally worked if all you exported from the `puppies` module was the `Puppy` model itself. Perhaps you changed that later when you started work on the seed script, and didn't notice it caused a _regression_ (breaking old code that used to work). This is the sort of thing we write tests for! :-)